uaclient/tests/test_data_types.py

```diff
@@ -229,6 +314,8 @@ def __init__(
         {"string": "nestedstring3", "integer": 7},
     ],
     "objlist_opt": None,
+    "enum_opt_list": None,
+    "enum_opt": None,
 }
 example_data_object_dict_with_optionals = {
```

Comment:
This object should include all optionals, including `enum_opt`

Reply:
I thought you were talking about line 317 and not 320

Reply:
Ah yeah my bad: github is annoying in not making it clear which line I clicked on to comment on :(